i18n/de/messages.de.xlf

```diff
@@ -0,0 +1,5105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+  <file source-language="en" datatype="plaintext" original="ng2.template">
+    <body>
+      <trans-unit id="70cd6a0cff2699d1c553c292ae76f41bb3b8e0de" datatype="html">
+        <source>Edit a resource</source>
+        <target>Eine Ressource bearbeiten</target>
```

Comment:
You could also just say "Ressource bearbeiten" if its like button or so I assume it looks better